src/main/resources/application.yaml

```diff
@@ -85,6 +85,7 @@ flyway:
   skip-migrations: ${FLYWAY_SKIP_MIGRATIONS}
 
 storage-blob-processing-delay-in-minutes: ${STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES}
+include-write-permission-in-storage-sas-token: ${INCLUDE_WRITE_PERMISSION_IN_STORAGE_SAS_TOKEN:true}
```

Comment:
this env var should be defined in `values.yaml` and that change should go first along with flux change. We can also remove the default here. Then this PR looks good to go

Reply:
Done. Flux change was always planned for this bit, now in https://github.com/hmcts/cnp-flux-config/pull/3088.